mingw-w64-git/PKGBUILD

```diff
@@ -187,7 +187,6 @@ package_git () {
 
   # Git Bash, Git CMD
   install -d -m755 $pkgdir$SHAREDIR
-  install -m755 ../git-for-windows.ico $pkgdir$SHAREDIR
```

Comment:
Ok, so we definitely keep the icon. However, would you still accept the change (kgybels/build-extra@c9ea4fa) as a pull request? It guarantees that the icon of the executable and the context menu entries are the same.